### DISCUSSION TITLE
Total Execution Time exceeds the filter value of 1 hour

### DISCUSSION THREAD

**Discussion by sebastianliebscher** [sidekiq/sidekiq#6468] [Q&A] [answered]
When looking at the Metrics dashboard in the web UI, I set the filter to 1 hour. Looking at my highest "Total Execution Time" with a value of 8200s (2.2h) it exceeds the dashboards filter value of 1h. How could this be interpreted? Any help or hint appreciated :)

**Comment by mperham** [comment, marked as answer]
If you have multiple threads working concurrently. Two threads can do two hours of work in one hour.
